1e/README.md

```diff
@@ -0,0 +1,50 @@
+# 1E
+
+## Overview
+
+The 1E Datadog Integration is an API-based integration that collects metrics from 1E products and forward them to your Datadog account. You can use these metrics to take advantage of the out-of-the-box dashboard or you can create your own visualizations.
+
+## Setup
+
+### Configuration
+
+Getting set up with the 1E Datadog integration is as simple as emailing a Datadog API key and region to the 1E Support team. There's no further configuration required.
+
+1. Go to your [Datadog Integrations page][1] and click on the 1E tile.
+
+2. Click the **Configuration** tab and click **Install Integration** at the bottom.
+
+3. Go to your [Datadog APIs page][2] and create an API key.
+
+4. Determine your Datadog API region by looking at your browser's address bar:
+
+   - If the domain name is `app.datadoghq.com`, your API region is `US`.
+   - If the domain name is `app.datadoghq.eu`, your API region is `EU`.
+
+5. Send an email to [1E support](mailto:support@1e.com) with the Datadog API key and region to activate your 1E Datadog integration.
```

Comment:
We recommend having the user input their api key somewhere on 1Es site since email isn't a secure way to send api keys.

Reply:
@emarsha94 Thanks for the suggestion. Can we consider this change in next PR or may be if these changes be released to staging environment for Dash showcase.

Reply:
Hi @Sid-1E, I'm a PM on the same team as @emarsha94. Unfortunately this is a hard blocker for our team - we can't allow for the transfer of API Keys over email as it poses a large security risk. The pattern we normally suggest is to have users enter their credentials on some kind of integration configuration page on your website. You could then pull those and use them in any subsequent calls to the Datadog API.

Reply:
We have discussed with our support team, and will ask customers to provide this detail via our secure 1E Support Portal. I have changed the set up steps accordingly.